packages/react/src/a11y/docs/overview.mdx

```diff
@@ -2,6 +2,7 @@
 title: A11y
 description: Utilities for improving accessibility.
 group: Foundations
+importNames: ['VisuallyHidden', 'ExternalLinkCallout']
```

Comment:
Just done some testing, and if you glob read and iterate over `packages/react/**/index.ts`, you can use `react-docgen-typescript` to grab the exports (`displayName`s) that it provides. We could do this in a new `generateComponentImports` file, `generateComponentProps` and then use it in to make the `import` statements